### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#438] [Esami & Esoneri M-Z]
## Traccia

<img width="624" alt="image" src="https://github.com/user-attachments/assets/5c4102b6-d2d4-45fc-b4d0-e6c38f8c820b">

**Comment by rimaout** [comment]
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/15af9f5e-b6b9-4a6e-9e9e-f9e0db065f4e">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/cb33334a-8532-4b83-bcfd-cd6ceb54807a)
